app/src/main/java/com/team22/soundary/feature/share/data/ShareApiService.kt

```diff
@@ -0,0 +1,14 @@
+package com.team22.soundary.feature.share.data
+
+import com.team22.soundary.feature.share.data.SearchTrackResponse
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface ShareApiService {
+    @GET("/api/v1/tracks")
+    fun requestMusicList(
+        @Query("platform") platform: String = "SPOTIFY",
+        @Query("query") query: String
+    ): Call<SearchTrackResponse>
```

Comment:
p4; coroutine 을 사용하여 call 객체를 받기보다 Response 로 바로 받는 것이 어떨까요?
```suggestion
    @GET("/api/v1/tracks")
    suspend fun requestMusicList(
        @Query("platform") platform: String = "SPOTIFY",
        @Query("query") query: String
    ): Response<SearchTrackResponse>
```

Reply:
이 부분을 수정하기 위해서는 학습이 필요할 것 같아 더 공부해서 다음 주 pr에 반영하도록 하겠습니다! 피드백 감사합니다!